service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/SendToGatekeeperController.java

```diff
@@ -29,4 +31,19 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(@RequestBod
 
         return respond(caseDetails);
     }
+
+    @PostMapping("/submitted")
+    public void handleSubmittedEvent(@RequestBody CallbackRequest callbackRequest) {
+        CaseData caseData = getCaseData(callbackRequest);
+
+        if (thirdPartyCreatedCase(caseData)) {
+            publishEvent(new NotifyManagedLAEvent(caseData));
+        }
+
+        publishEvent(new NotifyGatekeepersEvent(caseData));
+    }
+
+    private boolean thirdPartyCreatedCase(CaseData caseData) {
+        return (caseData.getOutsourcingPolicy() != null || caseData.getLocalAuthorityPolicy() != null);
```

Comment:
just caseData.getOutsourcingPolicy() !=null, localAuthorityPolicy is always present.

Reply:
we are doing https://tools.hmcts.net/jira/browse/FPLA-2670 as part of this story, how can we check if case is being managed by other LA if this is always present? outsourcing policy is only for EPS right? do you have some code from your story for this?

Reply:
cleared up confusion in chat - thanks